server/routes/auth/index.js

```diff
@@ -4,5 +4,9 @@ const router = require('express').Router();
 
 router.use('/google', require('./google'));
 router.use('/facebook', require('./facebook'));
+router.get('/logout', function(req, res){
+  req.logout();
+  res.redirect('/');
```

Comment:
I'd say this can be a success status (e.g. `204`) — let the client decide what to do once logged out, don't force a page reload. @cloworm @yuyingliu331